File: easy/1480_running_sum_of_1d_array/java/solution.java
import java.util.Arrays;

class solution {
    public int[] runningSum (int[] nums)
    {
        int[] result = new int[nums.length];
        int sum = 0;
        for (int i = 0; i< nums.length; i++)
        {
            sum += nums[i];
            result[i] = sum;
        }
        return result;
    }

    public static void main(String[] args) {
        solution solution = new solution();
        int[] nums = {1,2,3,4};
        System.out.println(Arrays.toString(solution.runningSum(nums)));
    }
}

File: easy/1480_running_sum_of_1d_array/csharp/solution.cs
class Solution{
    public int[] running_sum (int[] nums)
    {
        int[] result = [];
        int sum = 0;
        for (int i = 0; i<nums.Count; i++)
        {
            sum += nums[i];
            result.Add(sum);
        }
        return result;
    }
}

class Program {
    static void Main(string[] args) {
        Solution solution = new Solution();
        List<int> nums = [1,2,3,4];
        Console.WriteLine(string.Join(", ", solution.running_sum(nums)));
    }
}